docs/traffic-policy/actions/custom-response/index.mdx

```diff
@@ -0,0 +1,4 @@
+## Overview
+
+The **Custom Response** Traffic Policy action enables you to return hard-coded 
+responses back to the client that made a request to your endpoint.
```

Comment:
nit: "hard-coded responses" -> "a hard-coded response" (since it's to "a request")